src/component/variantPage/functionalPrediction/MutationAssesor.tsx

```diff
@@ -277,12 +275,11 @@ export default class MutationAssessor extends React.Component<
     }
 
     public render() {
-        let maContent: JSX.Element = (
-            <span className={`${annotationStyles['annotation-item-text']}`} />
-        );
+        let maContent: JSX.Element = <span />;
         const dataSource = (
-            <span className={functionalGroupsStyle['data-source']}>
-                Mutation Assessor
+            <span>
```

Comment:
if this is just text and <span> has no purpose but to contain, we could use React fragment instead.  so, `<>...</> `